.rubocop.yml

```diff
@@ -7,6 +7,8 @@ inherit_from:
 AllCops:
   TargetRubyVersion: 2.3
   DisplayCopNames: true
+  Exclude:
```

Comment:
This caused rubocop to explode again.  Could you please remove it?

Reply:
i think this snuck back in due to my rebase. Still haven't figured out what I did wrong there, but oh well.